material/base.html

```diff
@@ -86,8 +86,19 @@
       {% include "partials/header.html" %}
     {% endblock %}
     <div class="md-container">
-      {% set feature = config.extra.get("feature", {}) %}
-      {% if feature.tabs %}
+      <div class="md-banner"> 
+        <div class="md-banner__inner md-grid">
```

Comment:
Sure, up to now it's just a quick and dirty implementation which needs some cleanup but it will definitely be encapsulated in a partial.